### DISCUSSION TITLE
Timeout when attempting to install Hashnode app on repo

### DISCUSSION THREAD

**Discussion by nikfp** [community/community#38639] [Other Feature Feedback, Questions, & Ideas]
Not sure which discussion tag this needed to go to, so General it is. 

Hashnode says they have a feature to back blog content up to a GH repo, They have a link on their site that leads to: 

https://github.com/apps/hashnode/installations/new

However this URL times out. According to Hashnode this is an issue on the GH side and the page asks the user to contact GH support. See screenshot below. Opening devtools and reloading times out with a status code 502, indicating it is something on the GH side. 

Any direction would be greatly appreciated. 

![GH_timeout](https://user-images.githubusercontent.com/46945607/200740292-94b8c677-4697-4317-9177-5cdc14fd33b7.jpg)

**Comment by ossanna16** [comment]
Hi there @nikfp and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
